estimator/sim_data.py

```diff
@@ -0,0 +1,58 @@
+waypoint_dict = {
+    "Waypoint_SH"   : "Shelf",
```

Comment:
theoretically, more than one thing can be at the waypoint, but we can leave it for the real implementation